app/src/store/playground/types.ts

```diff
@@ -95,6 +95,8 @@ type PlaygroundInput = DatasetInput | ManualInput;
 export type ModelConfig = {
   provider: ModelProvider;
   modelName: string | null;
+  endpoint?: string | null;
```

Comment:
put this in the playground store right now, not persisted, persistence is pretty tough here since they are attahced to instances, of which there may be multiple. We can move it up to the credentials store, although then we still need to handle multiple instances and make a call on which instance to save. 

Any thoughts appreciated!

Reply:
sounds good tracking here
https://github.com/Arize-ai/phoenix/issues/5069